src/caret_analyze/value_objects/callback.py

```diff
@@ -563,8 +599,9 @@ def __init__(
         self._callback_name = callback_name
         self._symbol = symbol
         self._subscribe_topic_name = subscribe_topic_name
+        self._subscription_construction_order = subscription_construction_order
         self._service_name = service_name
-        self._publish_topic_names = publish_topic_names
+        self._publish_topics = publish_topics,
```

Comment:
```suggestion
        self._publish_topics = publish_topics
```

Please remove the comma to avoid being interpreted as a tuple.

Reply:
I fixed